es-app/src/scrapers/ScreenScraper.cpp

```diff
@@ -127,19 +127,30 @@ void screenscraper_generate_scraper_requests(const ScraperSearchParams& params,
 
 	path = ssConfig.getGameSearchUrl(params.game->getFileName());
 	auto& platforms = params.system->getPlatformIds();
+	std::vector<unsigned short> p_ids;
 
+	// Get the IDs of each platform from the ScreenScraper list
 	for (auto platformIt = platforms.cbegin(); platformIt != platforms.cend(); platformIt++)
 	{
 		auto mapIt = screenscraper_platformid_map.find(*platformIt);
 
 		if (mapIt != screenscraper_platformid_map.cend())
 		{
-			path += "&systemeid=";
-			path += HttpReq::urlEncode(std::to_string(mapIt->second));
+			p_ids.push_back(mapIt->second);
 		}else{
 			LOG(LogWarning) << "ScreenScraper: no support for platform " << getPlatformName(*platformIt);
```

Comment:
In the current version, if the platform is not found in `screenscraper_platformid_map` a warning is logged, but a request would still be generated without a system id.  In your updated version the request will no longer be generated.   Not sure if this is a regression or a bonus bug fix.

Reply:
You're right, it's a regression - I'll fix this.

Reply:
I ammended the modification - retaining the previous behavior.